client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -151,8 +151,12 @@ const UserPage = createClass({
 			return (_.deburr(brew.title).toLowerCase().includes(testString)) ||
 			(_.deburr(brew.description).toLowerCase().includes(testString));
 		}) : this.props.brews;
-		return _.groupBy(brewCollection, (brew)=>{
-			return (brew.published ? 'published' : 'private');
+		return _.groupBy(brewCollection, ({ collaborating, published })=>{
```

Comment:
The userPage has been changed with #1799 so this will need to be moved. Update this branch to master to see the changes.